.gitignore

```diff
@@ -18,3 +18,6 @@
 
 /config/secrets.yml
 /config/database.yml
```

Comment:
@prajaktar Why database and secrets are ignored and you are using a different version of them?

Reply:
Actually I just found it that now looking at the PR and surprised cause they were not committed.

Reply:
I had pushed a sample database and secrets file named as database-dist.yml and secrets-dist.yml and ignored secrets.yml and database.yml.

For example, in a team every developer may have different local database_name, username and password. So need to commit this file every time. To start your application, developer can create a copy of database-dist.yml as database.yml and edit the required information as per local credentials.

With docker I suppose this case will be solved as every developer will use the common mysql image.

Reply:
Exactly, you got it, it is the idea of using docker, to standardize the setup and dev environment even making it equal or close to production.